### PR TITLE
Create the release on Travis CI to avoid release build issues

### LINKED ISSUE #1217: Openlayers error in admin interface
I get an error when a page of admin interface contains an Openlayers map : 

Uncaught TypeError: undefined is not a function
on following line
new OpenLayers.Layer.OSM("OSM"),
    'POLYGON((710189.563857429 5896129.24645693,712024.052614936 5825195.68388646,797633.524960828 5824584.18747307,796410.532424497 5896129.24626647,710189.563857429 5896129.24645693))'
);

The map does not display.

### PR DESCRIPTION
**Pull request by sbrunner**
To definitively fix #1217

### CODE REVIEW

File: doc/developer/build_release.rst [hunk 1]
```diff
@@ -203,3 +166,8 @@ Push your changes:
 
     git push origin <version>
     git push origin <release>
+
+.. note::
+
+   When you push a tag with the pattern `^[0-9].[0-9]+.[0-9]$` a new release
+   will automatialy be create on Travis CI.
```
Comment (asaunier) [inline doc/developer/build_release.rst]: create => created

File: travis/deploy.sh [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+if [[ $TRAVIS_BRANCH =~ ^(master|[0-9].[0-9])$ ]] && [ $TRAVIS_PULL_REQUEST == false ]
+then
+    DEPLOY=true
+fi
+
+if [[ $TRAVIS_TAG =~ ^[0-9].[0-9]+.[0-9]$ ]]
+then
+    if [ $TRAVIS_TAG != $(python setup.py -V) ]
+    then
+        echo "The version don't corresponds"
```
Comment (asaunier) [inline travis/deploy.sh]: don't corresponds => doesn't correspond (or doesn't match)

Comment (sbrunner) [inline travis/deploy.sh]: => The tag name doesn't match with the egg version.

File: doc/developer/build_release.rst [hunk 3]
```diff
@@ -203,3 +166,8 @@ Push your changes:
 
     git push origin <version>
     git push origin <release>
+
+.. note::
+
+   When you push a tag with the pattern `^[0-9].[0-9]+.[0-9]$` a new release
+   will automatialy be created on Travis CI.
```
Comment (asaunier) [inline doc/developer/build_release.rst]: Oups, I have missed this one:
automatialy => automatically

**Pull request merged**